gnome3_*.libgdata: add `nss` dependency

nckx opened the pull request:
Fixes the following build error:

```
  CCLD     gdata/libgdata.la
/nix/store/5kdjp8200hazaydx0dmwn5qghqkyi3py-binutils-2.23.1/bin/ld: cannot find -lssl3
/nix/store/5kdjp8200hazaydx0dmwn5qghqkyi3py-binutils-2.23.1/bin/ld: cannot find -lsmime3
/nix/store/5kdjp8200hazaydx0dmwn5qghqkyi3py-binutils-2.23.1/bin/ld: cannot find -lnss3
/nix/store/5kdjp8200hazaydx0dmwn5qghqkyi3py-binutils-2.23.1/bin/ld: cannot find -lnssutil3
collect2: error: ld returned 1 exit status
```

cc @lethalman.

nckx commented:
:+1:

lethalman commented:
That's probably not due to libgdata but to dependencies having broken .la files.

nckx commented:
Ugh. That's what I feared. Thanks.

nckx committed:
liboauth: add `nss3` path to liboauth.la

Fixes the following libgdata build error:
```
  CCLD     gdata/libgdata.la
/nix/store/5kdjp8200hazaydx0dmwn5qghqkyi3py-binutils-2.23.1/bin/ld: cannot find -lssl3
/nix/store/5kdjp8200hazaydx0dmwn5qghqkyi3py-binutils-2.23.1/bin/ld: cannot find -lsmime3
/nix/store/5kdjp8200hazaydx0dmwn5qghqkyi3py-binutils-2.23.1/bin/ld: cannot find -lnss3
/nix/store/5kdjp8200hazaydx0dmwn5qghqkyi3py-binutils-2.23.1/bin/ld: cannot find -lnssutil3
collect2: error: ld returned 1 exit status
```

nckx commented:
@lethalman: V2.

lethalman commented:
Better thanks :)

vcunat commented:
This was needed even on 15.09. http://hydra.nixos.org/build/29135622/nixlog/1/tail a1c74de
